notification/pom.xml

```
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
-		<groupId>com.scaler</groupId>
-		<artifactId>public-grievance-management</artifactId>
-		<version>0.0.1-SNAPSHOT</version>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>3.1.5</version>
```

```suggestion
		<groupId>com.scaler</groupId>
		<artifactId>public-grievance-management</artifactId>
		<version>0.0.1-SNAPSHOT</version>
```
To enable the Spring Boot application to function within a multi-module architecture, it is essential to specify the groupId, artifactId, and version in the pom.xml file for each module app as the corresponding groupId, artifactId, and version for the parent pom.xml.